### DISCUSSION TITLE
Github failure message related to cask

### DISCUSSION THREAD

**Discussion by codingbutstillalive** [Homebrew/discussions#612] [Casks]
Dear fellows,

I just had a fresh install of homebrew. When I run 

`brew install --cask rectangle`

it gives me the following output:

**_Error: Failure while executing; `git clone https://github.com/Homebrew/homebrew-cask /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask` exited with 128._**

How to fix that? :)

**Comment by SMillerDev** [comment]
What happens when you run `brew tap homebrew/cask`? Please post the full output here.

**Comment by codingbutstillalive** [comment]
==> Tapping homebrew/cask
Cloning into '/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask'...
remote: Enumerating objects: 12, done.
remote: Counting objects: 100% (12/12), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 545925 (delta 4), reused 6 (delta 4), pack-reused 545913
Receiving objects: 100% (545925/545925), 240.13 MiB | 13.00 MiB/s, done.
Resolving deltas: 100% (385419/385419), done.
Updating files: 100% (3892/3892), done.
Tapped 3796 casks (3,917 files, 257.7MB).

**Reply by codingbutstillalive** [reply]
After executing this command, I tried again to install "rectangle" and this time it worked.
